lib/server.js

```diff
@@ -1151,10 +1159,13 @@ function MQTTServer(adapter) {
             if (str.startsWith('{"POWER":"ON"}')) return;
             if (str.startsWith('{"POWER":"OFF"}')) return;
 
-
             if (parts[0] === 'stat') {
                 try {
-                    checkData(client, packet.topic, NO_PREFIX, JSON.parse(val));
+                    checkData(client, packet.topic, NO_PREFIX, JSON.parse(val));			
+                    if (val.includes('Script')) {
```

Comment:
Should be here `val.includes('Script_')` too as above?

Reply:
No, Tasmota sends a json e.g. {"script": ">soll=20"}. I turn it into an object script_soll with the value 20. When sending script_soll with the value 30 to Tasmota it has to be converted back to {"script": ">soll=30"}.
The values can be changed directly in the new Tasmota scripting language. To remain downward compatible, the Script object with the value ">soll=20" is also created. You could also write ">=30" in the Script object, but this is complicated.

Reply:
Look here:
[https://forum.iobroker.net/topic/26237/beca-thermostate-einbinden-firmware-mod/48](url)